packages/plugins/parser-cache/test/parser-cache.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { buildSchema, DocumentNode, parse } from 'graphql';
+import { buildSchema, DocumentNode, GraphQLError, parse } from 'graphql';
 import { assertSingleExecutionValue, createTestkit } from '@envelop/testing';
 import { useParserCache } from '../src/index.js';
 import { Plugin } from '@envelop/types';
-import lru from 'tiny-lru';
+import lru from 'lru-cache';
```

Comment:
Since now it’s a class it’s better to change name to 
```suggestion
import LRU from 'lru-cache';
```